docs/style-guide.rst

```diff
@@ -880,10 +880,11 @@ Order of Layout
 
 Layout contract elements in the following order:
 
-1. Import statements
-2. Interfaces
-3. Libraries
-4. Contracts
+1. Pragma statements
+2. Import statements
```

Comment:
Should pragma come before import? Note, most of the pragma setting stay alive cross files if import is after the pragma. Do we have that mentioned somewhere?

Reply:
oh wow, I wasn't aware of that.